ui/pages/confirmations/components/confirm/info/approve/approve-details/approve-details.test.tsx

```diff
@@ -27,4 +27,14 @@ describe('<ApproveDetails />', () => {
     );
     expect(container).toMatchSnapshot();
   });
+
+  it('does not render SigningInWithRow for setApprovalForAll', () => {
+    const state = mockState;
+    const mockStore = configureMockStore(middleware)(state);
+    const { queryByText } = renderWithConfirmContextProvider(
+      <ApproveDetails isSetApprovalForAll />,
+      mockStore,
+    );
+    expect(queryByText('signingInWith')).toBeNull();
+  });
```

Comment:
I think it would be helpful to add the contra test for this 
- renders SigningInWithRow for setApprovalForAll

Reply:
Thanks for the review :pray: 
I had to change the solution after discussing it with Barbara and Jyoti, so I would greatly appreciate it if you could review it again.